composer.json

```diff
@@ -31,9 +31,8 @@
     "oat-sa/generis" : ">=15.22",
     "oat-sa/tao-core" : ">=50.24.6",
     "oat-sa/extension-tao-item" : ">=11.11.2",
-    "oat-sa/extension-tao-outcomeui" : ">=10.0.0",
     "oat-sa/extension-tao-itemqti" : ">=28.22.0",
-    "oat-sa/extension-tao-testqti" : ">=41.22.2",
+    "oat-sa/extension-tao-testqti" : ">=47.2.3",
```

Comment:
this dependency was released as 47.3.0
https://github.com/oat-sa/extension-tao-testqti/pull/2398#issuecomment-1711345180
@yaraslau-kavaliou please be careful with dependencies